aio/aio-proxy/aio_proxy/doc/open-api.yml

```diff
@@ -0,0 +1,381 @@
+      parameters:
+        - name: q
+          in: query
+          description: Termes de la recherche (dénomination et/ou adresse)
+          required: true
+          schema:
+            type: string
+        - name: code_postal
+          in: query
+          description: <a href=https://www.sirene.fr/sirene/public/variable/codpos>Code postal en 5 chiffres</a>
+          required: false
+          schema:
+            type: string
+        - name: activite_principale_entreprise
+          in: query
+          description: <a href=https://www.sirene.fr/sirene/public/variable/apet700-rev2>Le code NAF ou code APE, un code d'activité suivant la nomenclature de l'INSEE</a>
+          required: false
+          schema:
+            type: string
+        - name: is_entrepreneur_individuel
+          in: query
+          description: Uniquement les entreprises individuelles
+          required: false
+          schema:
+            type: string
+            enum:
+             - "yes"
+             - "no"
+        - name: tranche_effectif_salarie_entreprise
+          in: query
+          description: <a href=https://www.sirene.fr/sirene/public/variable/tefen>Tranche d'effectif salarié de l'entreprise</a>
+          required: false
+          schema:
+            type: string
+        - name: page
+          in: query
+          description: Le numéro de la page à retourner
+          required: false
+          schema:
+            type: integer
+            default: 1
+        - name: per_page
+          in: query
+          description: Le nombre de résultats par page
+          required: false
+          schema:
+            type: integer
+            default: 10
+      responses:
+        "200":
+          description: La liste des entreprises correspondantes à la recherche.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  results:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        siren:
+                          type: integer
+                          example: 111111111
+                          description: le numéro unique de l'entreprise
+                        siret_siege:
+                          type: integer
+                          example: 11111111111111
+                          description: le numéro unique de l'établissement siège
+                        date_creation_siege:
+                          type: string
+                          format: date-time
+                          description: Date de création du siège
+                        tranche_effectif_salarie_siege:
+                          type: string
+                        activite_principale_registre_metier:
+                          type: string
+                        numero_voie:
+                          type: number
```

Comment:
Are you sure ? What about "23 bis rue de ségur" ? `numero_voie` is `23` or `23 bis`. I think there a specific column on sirene database for this info that I do not find in this swagger, it is something like : `indicederepetition`

Reply:
I do not use the `indicederepetition` column, so I typed it as string just in case.